### DISCUSSION TITLE
AMD FSR 2.0 for max FPS!!

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [IrisShaders/Iris#1436] [Ideas]
Honestly not much to say here. Would it be a huge project to get AMD FSR 2.0 working with Iris Shaders to squeeze extra performance out of them for us peasants on older GPUs? I think that'd be amazing!

**Comment by IMS212** [comment]
FSR 1.0 should be possible, but no one has the time to look into it. FSR 2.0 is not publicly released outside of one game currently.
